trace_test.go

```diff
@@ -106,6 +115,18 @@ func TestGetFields(t *testing.T) {
 	assert.Equal(t, fields, GetFields(e))
 }
 
+func TestStackTraceReport(t *testing.T) {
+	testErr := fmt.Errorf("description")
+	assert.Equal(t, Traces{}, StackTraceReport(testErr))
+
+	err := Wrap(testErr, "user message")
+	stackTrace := StackTraceReport(err)
+	assert.Len(t, stackTrace, 3)
+	assert.Equal(t, stackTrace[0].Func, "github.com/gravitational/trace.TestStackTraceReport")
+	assert.Equal(t, stackTrace[1].Func, "testing.tRunner")
+	assert.Equal(t, stackTrace[2].Func, "runtime.goexit")
```

Comment:
can we avoid asserting on these implementation details? probably best to create a stack trace with a few levels we control and only assert on those